pkg/security/probe/probe_windows.go

```diff
@@ -149,20 +165,25 @@ func NewProbe(config *config.Config, opts Opts) (*Probe, error) {
 		cancelFnc:            cancel,
 		StatsdClient:         opts.StatsdClient,
 		discarderRateLimiter: rate.NewLimiter(rate.Every(time.Second/5), 100),
-		event:                &model.Event{},
 		PlatformProbe: PlatformProbe{
 			onStart: make(chan *procmon.ProcessStartNotification),
 			onStop:  make(chan *procmon.ProcessStopNotification),
 		},
 	}
-	resolvers, err := resolvers.NewResolvers(config, p.StatsdClient)
+
+	p.scrubber = procutil.NewDefaultDataScrubber()
```

Comment:
How come Linux doesn't need to scrub sensitive words?

Reply:
it does : https://github.com/DataDog/datadog-agent/blob/main/pkg/security/probe/probe_linux.go#L1627